spec/operators/merge-map-spec.js

```diff
@@ -139,10 +139,15 @@ describe('Observable.prototype.mergeMap()', function () {
     var values = {i: 'foo', j: 'bar', k: 'baz', l: 'qux'};
     var e1 =    hot('-a-------b-------c-------d-------|');
     var inner = cold('----i---j---k---l---|', values);
+    var innersubs =['-^-------------------!',
+                    '---------^-------------------!',
+                    '-----------------^-------------------!',
+                    '-------------------------^-------------------!'];
     var expected =  '-----i---j---(ki)(lj)(ki)(lj)(ki)(lj)k---l---|';
 
     expectObservable(e1.mergeMap(function(value) { return inner; }))
       .toBe(expected, values);
+    expectSubscriptions(inner.subscriptions).toBe(innersubs);
   });
```

Comment:
Nice.

Can you add something to [the docs](https://github.com/ReactiveX/RxJS/blob/master/doc/writing-marble-tests.md) about this?